index.js

```diff
@@ -25,13 +25,18 @@ export async function setupPlugin({ config, global }) {
         throw new RetryError('Service is down, retry later')
     }
 
+    const eventNames = (config.eventsToSend || '').split(',').filter(String)
```

Comment:
I don't think `.filter(String)` does anything? I think it should be `.filter(Boolean)` if you want to filter out non-empty elements.

Reply:
@Twixes So I will be the first to admit I am not the best javascript dev, but I did test this out and it does filter out the empty elements. I have tested it with the boolean way and that does work as well. Want me to do the boolean way??

Reply:
Oh, you're right, I forgot JS does a conversion to boolean here anyway. No need to change then.